app/build/plugins/image/index.js

```diff
@@ -11,43 +13,58 @@ function render ($, callback, options) {
   // Process 5 images concurrently
   eachEl($, 'img', function(el, next){
 
-    var src = $(el).attr('src');
+    // Decode any doubly-encoded ampersands in the image src
+    var src = decodeAmpersands($(el).attr('src'));
```

Comment:
Can we move this to the Transformer module? Doing so will apply this fix to the thumbnail generator as well as the image cacher

Reply:
I guess decodeAmpersands is needed here to parse the SRC correctly

Reply:
Right, it needs to exist here in order to parse the URL for the `static` param.